c++/src/capnp/membrane.c++

```diff
@@ -311,7 +311,7 @@ public:
 
 private:
   kj::Own<CallContextHook> inner;
-  kj::Own<MembranePolicy> policy;
+  kj::Rc<MembranePolicy> policy;
```

Comment:
In this case we increment the refcount THREE times for every MembraneCallContext, only to be decremented all together in the destructor.